### PR TITLE
Fix aad graph endpoint for SQL server and managed instance APIs

### PR DESCRIPTION
**Pull request by amolagar5**
## Description

Move endpoint used in SQL Server and SQL Instance from AD Graph to MS Graph

### CODE REVIEW

**Review by strehan1993 — APPROVED** [review]

**Review by ericshape — DISMISSED** [review]
LGTM

**Comment by dingmeng-xue** [comment]
Do you expect this feature go into Ignite release?

File: tools/TestFx/TestEnvironmentFactory.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ static TestEnvironmentFactory()
 
         internal static TestEnvironment BuildTestFxEnvironment()
         {
-            _environment.SetRecordedEnvironmentVariables();
+            _environment.SetEnvironmentVariables();
```
Comment (dingmeng-xue) [inline tools/TestFx/TestEnvironmentFactory.cs:33]: why the files under TestFx need to be changed?

**Review by dingmeng-xue — CHANGES_REQUESTED** [review]

File: tools/TestFx/TestEnvironmentFactory.cs (hunk 1 above)
Comment (vidai-msft) [reply tools/TestFx/TestEnvironmentFactory.cs:33]: For batch tests, the TestEnvironment object is initiated only once. However, the environment variables (set by SetEnvironmentVariables()) need to be re-populated for each test case. Previously, only Playback mode got it re-populated, so it caused SubscriptionId missing in json file during recording.

Comment (amolagar5) [reply tools/TestFx/TestEnvironmentFactory.cs:33]: Thanks @vidai-msft for fixing this.

**Comment by amolagar5** [comment]
> Do you expect this feature go into Ignite release?

Yes. sooner we can release this, better it is. 
We have 2 CRIs due to this so far already.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**